packages/sanity/src/core/components/userAvatar/UserAvatar.tsx

```diff
@@ -93,7 +93,6 @@ const StaticUserAvatar = forwardRef(function StaticUserAvatar(
       size={typeof size === 'string' ? LEGACY_TO_UI_AVATAR_SIZES[size] : size}
       status={status}
       title={user?.displayName}
-      {...restProps}
```

Comment:
Removed as this wasn't passing any valid props, which was causing warnings in development.